src/sentry/features/manager.py

```diff
@@ -139,6 +139,18 @@ def has_for_batch(
 FLAGPOLE_OPTION_PREFIX = "feature"
 
 
+def capture(fn):
+    @functools.wraps(fn)
+    def wrapper(
+        self, name: str, *args: Any, skip_entity: bool | None = False, **kwargs: Any
+    ) -> bool:
+        result = fn(self, name, *args, skip_entity=skip_entity, **kwargs)
+        sentry_sdk.feature_flags.add_feature_flag(name, result)
+        return result
+
+    return wrapper
```

Comment:
this decorator breaks typing (by turning anything it decorates into `Any`) -- please make sure this is properly typed -- might be of help: 
- https://youtu.be/fwZoxWyMGM8 (decorator typing PEP 612)